src/tests/integration/test_generic_db_operations.py

```diff
@@ -636,10 +637,63 @@ def test_sqlcmd_tables(ip_with_dynamic_db, request):
 @pytest.mark.parametrize("ip_with_dynamic_db", ALL_DATABASES)
 def test_sql_query(ip_with_dynamic_db, cell, request):
     ip_with_dynamic_db = request.getfixturevalue(ip_with_dynamic_db)
+    ip_with_dynamic_db.run_cell(
+        """
+    %%sql sqlite://
+    CREATE TABLE test_numbers (value_one, value_two);
+    INSERT INTO test_numbers VALUES (0, 1);
+    INSERT INTO test_numbers VALUES (0, 0);
+    INSERT INTO test_numbers VALUES (5, 2);
+    INSERT INTO test_numbers VALUES (6, 3);
+    """
+    )
     ip_with_dynamic_db.run_cell(
         """%%sql --save subset --no-execute
 SELECT * FROM numbers WHERE 1=0
 """
     )
     out = ip_with_dynamic_db.run_cell(cell)
     assert out.error_in_exec is None
+
+
+@pytest.mark.parametrize(
+    "ip_with_dynamic_db",
+    [
+        ("ip_with_postgreSQL"),
+        ("ip_with_mySQL"),
+        ("ip_with_mariaDB"),
+        ("ip_with_SQLite"),
+        ("ip_with_duckDB"),
+        pytest.param(
+            "ip_with_MSSQL",
+            marks=pytest.mark.xfail(reason="issue in capturing the syntax error"),
```

Comment:
What's the issue? should we open a new issue for it?

Reply:
I'm having trouble setting up MSSQL locally. and not sure what error is throwing for syntax errors

Reply:
Solved the issue. MSSQL case is also passing now